developer_manual/app/custom-cache-backend.rst

```diff
@@ -0,0 +1,246 @@
+===============================
+Create Custom Filesystem Caches
+===============================
+
+.. sectionauthor:: Matthew Setter <msetter@owncloud.com>
+
+The metadata cache in ownCloud can be overwritten by a storage class backend which implements the following methods:
+
++---------------------------------------------+---------------------------------------------------+
+| Method                                      | Description                                       |
++=============================================+===================================================+
+| ``getCache($path = '', $storage = null)``   | For overwriting the cache itself.                 |
+| ``getScanner($path = '', $storage = null)`` | For overwriting the meta data scanning behavior.  |
+| ``getWatcher($path = '', $storage = null)`` | For overwriting the behavior of checking for      |
+|                                             | external changes.                                 |
++---------------------------------------------+---------------------------------------------------+
+
+.. note: 
+   The ``$storage`` argument should be passed to the created cache, scanner, or watcher and default to the current storage. The ``$path`` argument can be used as a hint to where the cache, scanner, or watcher is going to be used for.
```

Comment:
I don't understand the meaning of "hint to where the cache...is going to be used for".
Should it be "hint to what the cache...is going to be used for" or "hint to where the cache...is going to be used" or?